### DISCUSSION TITLE
How to reduce the noise for each release?

### DISCUSSION THREAD

**Discussion by frederikhors** [jycouet/kitql#44] [General]
I signed up on "All Activity" but with each release I get lots of emails.

Is there a way to reduce the amount of emails that arrive perhaps by reducing the release steps?

![image](https://user-images.githubusercontent.com/41120635/155501549-830c09ca-619d-4947-a815-b4667f7017ac.png)

**Comment by jycouet** [comment]
It's annoying me as well.
I didn't find anything on repo level.

You can try maybe:
![image](https://user-images.githubusercontent.com/5312607/155504085-fa917fe7-6fc9-468d-bae0-4ee4b75f2625.png)

**Reply by frederikhors** [reply]
Yeah, but I need to know when a new release is there! LOL 😄 I think we can reduce email avoiding new PR on each release. But now let's focus on code not on this.

**Reply by jycouet** [reply]
haha, but you don't need to know the release, since you have the PR that will trigger the release you are good.
Once the PR release merged => It's published ^^.

I'm trying this conf right now. let's see :)
